README.md

```diff
@@ -2,6 +2,8 @@
 
 Very basic Puppeteer project, with example code for several test scenarios.
 
+Worst project ever! You should quit.
```

Comment:
I feel like the exclamation point is a bit too much. Change it to a period, perhaps?